pkg/utils/chart.go

```diff
@@ -328,3 +344,23 @@ func getReleaseName(hr *appsapi.HelmRelease) string {
 	}
 	return releaseName
 }
+
+func replaceCRD(cfg *action.Configuration, targetChart *chart.Chart) error {
+	queue := []*chart.Chart{targetChart}
+	for _, c := range queue {
+		for _, crd := range c.CRDObjects() {
+			crdResource, err := cfg.KubeClient.Build(bytes.NewBuffer(crd.File.Data), true)
+			if err != nil {
+				return err
+			}
+			res, err := cfg.KubeClient.Update(crdResource, crdResource, true)
+			if err != nil {
```

Comment:
I'd prefer to have the errors aggregated. Please refer to method `NewAggregate` in package `utilerrors "k8s.io/apimachinery/pkg/util/errors"`.